generated/nimodinst/attributes.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
```

Comment:
This file is gone in master, don't re-add.

Reply:
Fixed